dokka-subprojects/core/src/main/kotlin/org/jetbrains/dokka/links/DRI.kt

```diff
@@ -139,6 +167,11 @@ public data class PointingToCallableParameters(val parameterIndex: Int) : DriTar
     override fun toString(): String = "PointingToCallableParameters($parameterIndex)"
 }
 
+@ExperimentalDokkaApi
+public data class PointingToContextParameters(val parameterIndex: Int) : DriTarget() {
+    override fun toString(): String = "PointingToContextParameters($parameterIndex)"
+}
+
 public fun DriTarget.nextTarget(): DriTarget = when (this) {
     is PointingToGenericParameters -> PointingToGenericParameters(this.parameterIndex + 1)
     is PointingToCallableParameters -> PointingToCallableParameters(this.parameterIndex + 1)
```

Comment:
Should we add `PointingToContextParameters` here?

Reply:
I changed it just to keep the contract of the public function.